vtex-apps/b2b-quotes
# Update pt.json

3 missing rows now translated to PT-BR

#### What problem is this solving?
Complete translation into portuguese BR.



#### How to test it?
Navigate, create quotes, configure parameters... etc. to see the messages.



adberaldob2b

[Workspace](Link goes here!)

#### Screenshots or example usage:



#### Describe alternatives you've considered, if any.



#### Related to / Depends on



#### How does this PR make you feel? [:link:](http://giphy.com/)



![](put .gif link here - can be found under "advanced" on giphy)
